src/redux/books/books.js

```diff
@@ -0,0 +1,24 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+export default function booksReducer(state = [], action) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.book];
+    case REMOVE_BOOK:
+      return [...state.filter((book) => book.id !== action.bookId)];
+    default:
+      return state;
+  }
+}
+
+export const addBook = (book) => ({
+  type: ADD_BOOK,
+  book,
+});
+
+export const removeBook = (bookId) => ({
+  type: REMOVE_BOOK,
+  bookId,
+
+});
```

Comment:
- [ ] Kindly fix the following issues that have been raised on the console side to make the project run successfully.

![image](https://user-images.githubusercontent.com/55635977/173627645-2c5d4810-8170-48c6-83e7-dc535836b13b.png)